source/uwp/Visualizer/ViewModel/GenericDocumentViewModel.cs

```diff
@@ -117,15 +120,58 @@ protected async Task LoadFromFileAsync(IStorageFile file, string token, bool ass
             }
         }
 
+        private async Task CreateMessageDialog(string text)
+        {
+            var dontWait = new MessageDialog(text);
```

Comment:
>dontWait [](start = 16, length = 8)

Why is this called dontWait?

Reply:
that was the original name, I thought there was a reason behind so I just kept the name. I'll change it to messageDialog


---
In reply to: [389910458](https://github.com/microsoft/AdaptiveCards/pull/3825#discussion_r389910458) [](ancestors = 389910458)

Reply:
Ah, i see. Looks like before we were not waiting on the async call, and the dontwait variable would have been the async action we were explicitly not waiting on. Now it's a message dialog, so changing the name makes sense. 

But now I'm wondering: why the change from not waiting to waiting?

---
In reply to: [389913301](https://github.com/microsoft/AdaptiveCards/pull/3825#discussion_r389913301) [](ancestors = 389913301,389910458)

Reply:
There's no regular Show method, there's only the ShowAsync method

---
In reply to: [389916462](https://github.com/microsoft/AdaptiveCards/pull/3825#discussion_r389916462) [](ancestors = 389916462,389913301,389910458)

Reply:
Right, but previously we didn't "await" the result from ShowAsync. In the old code, we put the IAsyncOperation in a variable called dontWait, and didn't "await" it. In the new code we await the IAsyncOperation.

---
In reply to: [389920998](https://github.com/microsoft/AdaptiveCards/pull/3825#discussion_r389920998) [](ancestors = 389920998,389916462,389913301,389910458)

Reply:
I didn't know that you could capture an IAsyncOperation with a variable. changed it to not wait again

---
In reply to: [389924506](https://github.com/microsoft/AdaptiveCards/pull/3825#discussion_r389924506) [](ancestors = 389924506,389920998,389916462,389913301,389910458)